packages/account/src/Sections/Verification/Helpers/verification.js

```diff
@@ -1,19 +1,25 @@
-export const populateVerificationStatus = account_status => {
-    const { attempts, document, identity, needs_verification } = account_status.authentication;
+const populateVerificationStatus = account_status => {
+    const { attempts, document, identity, needs_verification } = {
+        ...account_status?.authentication,
+        attempts: account_status?.authentication?.attempts || {},
+        needs_verification: account_status?.authentication?.needs_verification || {},
+    };
     const has_poa = !(document && document.status === 'none');
     const has_poi = !(identity && identity.status === 'none');
     const needs_poa = needs_verification.length && needs_verification.includes('document');
     const needs_poi = needs_verification.length && needs_verification.includes('identity');
 
-    const allow_document_upload = account_status.status.some(status => status === 'allow_document_upload');
-    const allow_poi_resubmission = account_status.status.some(status => status === 'allow_poi_resubmission');
-    const is_idv_disallowed = account_status.status.some(status => status === 'idv_disallowed');
+    const allow_document_upload = account_status?.status?.some(status => status === 'allow_document_upload');
+    const allow_poi_resubmission = account_status?.status?.some(status => status === 'allow_poi_resubmission');
+    const is_idv_disallowed = account_status?.status?.some(status => status === 'idv_disallowed');
```

Comment:
Just a proposal: we can create a function for similar status accessing

Reply:
Done

Reply:
Done